aiidalab/utils.py

```diff
@@ -124,7 +124,7 @@ def fulfills(self, requirement):
         )
 
 
-@cached(cache=TTLCache(maxsize=32, ttl=60))
+@cached(cache=TTLCache(maxsize=32, ttl=5))
```

Comment:
Instead of reducing the cache ttl substantially, can we maybe clear it explicitly instead? Here is a minimal example how to do so:
```python
from cachetools import TTLCache, cached
from time import sleep

_TTL_CACHE=TTLCache(maxsize=32, ttl=60)

@cached(cache=_TTL_CACHE)
def cached_square_func(a):
    print("calculating...")
    sleep(1)
    return a * a


print(cached_square_func(2))
print(cached_square_func(2))
_TTL_CACHE.clear()
print(cached_square_func(2))
```
Note, there might be a better way to do this, I only spend a few minutes on this.

Reply:
Thanks, @csadorf, I just tried and it seems to work. Also, I did a bit of research in Google and the way you proposed seems to be the one people suggest.